packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentTitle.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { paymentInfo } from '../../../__mocks__/ExternalRegistry.mock';
+import { notificationToFe, payments } from '../../../__mocks__/NotificationDetail.mock';
+import { PaymentsData } from '../../../models';
+import { initLocalizationForTest, render } from '../../../test-utils';
+import { getF24Payments, getPagoPaF24Payments, populatePaymentsPagoPaF24 } from '../../../utility';
+import NotificationPaymentTitle from '../NotificationPaymentTitle';
+
+describe('NotificationPaymentTite component', () => {
+  const paymentsData: PaymentsData = {
+    pagoPaF24: populatePaymentsPagoPaF24(
+      notificationToFe.timeline,
+      getPagoPaF24Payments(payments, 0),
+      paymentInfo
+    ),
+    f24Only: getF24Payments(payments, 0),
+  };
+
+  beforeAll(() => {
+    initLocalizationForTest();
+  });
+
+  it('render component - multi payments', () => {
+    const { container, getByTestId } = render(
+      <NotificationPaymentTitle
+        pagoPaF24={paymentsData.pagoPaF24}
+        f24Only={[]}
+        landingSiteUrl="https://www.mocked-url.com"
+        handleTrackEventFn={() => {}}
+      />
+    );
+
+    expect(container).toHaveTextContent(
+      'notifiche - detail.payment.subtitle notifiche - detail.payment.how'
+    );
+    const faq = getByTestId('faqNotificationCosts');
+    expect(faq).toBeInTheDocument();
+  });
+
+  it('render component - one payment', () => {
+    const { container, getByTestId } = render(
+      <NotificationPaymentTitle
+        pagoPaF24={[paymentsData.pagoPaF24[0]]}
+        f24Only={[]}
+        landingSiteUrl="https://www.mocked-url.com"
+        handleTrackEventFn={() => {}}
+      />
+    );
+
+    expect(container).toHaveTextContent(
+      'notifiche - detail.payment.single-payment-subtitle notifiche - detail.payment.how'
+    );
+    const faq = getByTestId('faqNotificationCosts');
+    expect(faq).toBeInTheDocument();
+  });
+
+  it('render component - multi payments whit no f24', () => {
```

Comment:
I think this test is identical to the first one.

Reply:
You are right! I have changed the first test

Reply:
great